### PR TITLE
Tickets/dm 31995

### PR DESCRIPTION
**Pull request by patrickingraham**

### CODE REVIEW

File: .githooks/pre-commit [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```
Comment (tribeiro) [inline .githooks/pre-commit:1]: I think we stopped using the githook in favor of the pre-commit.. The package should already be configure with it (https://github.com/lsst-ts/ts_externalscripts/blob/develop/.pre-commit-config.yaml) though, I must admit I am not really sure how they work.. maybe @r-owen can wave in here..

**Review by tribeiro — APPROVED** [review]
Looks good, but please see my comment about the hook..

File: .githooks/pre-commit (hunk 1 above)
Comment (r-owen) [reply .githooks/pre-commit:1]: I'm afraid that issue is still present. I wish I'd seen this before I merged my update. I'll make a small branch for this.

Comment (r-owen) [reply .githooks/pre-commit:1]: Good catch. I fixed this (as an extra commit after merging my ticket).

**Pull request merged**